MonMaraicher/Model/Farmer/Farmers.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Produceurs.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 10/04/2024.
+//
+
+import Foundation
+
+struct Farmers: Decodable, Hashable {
+    let nbTotal: String
```

Comment:
nbTotal of String? a number is a number not a string

Reply:
For this property, I don't know how to do it yet to convert it into Int